sqlglot/dialects/dialect.py

```diff
@@ -99,44 +99,58 @@ def get_start_end(token_type: TokenType) -> t.Tuple[t.Optional[str], t.Optional[
                 (None, None),
             )
 
-        klass.bit_start, klass.bit_end = get_start_end(TokenType.BIT_STRING)
-        klass.hex_start, klass.hex_end = get_start_end(TokenType.HEX_STRING)
-        klass.byte_start, klass.byte_end = get_start_end(TokenType.BYTE_STRING)
-        klass.raw_start, klass.raw_end = get_start_end(TokenType.RAW_STRING)
+        klass.BIT_START, klass.BIT_END = get_start_end(TokenType.BIT_STRING)
+        klass.HEX_START, klass.HEX_END = get_start_end(TokenType.HEX_STRING)
+        klass.BYTE_START, klass.BYTE_END = get_start_end(TokenType.BYTE_STRING)
+        klass.RAW_START, klass.RAW_END = get_start_end(TokenType.RAW_STRING)
 
-        klass.tokenizer_class.identifiers_can_start_with_digit = (
-            klass.identifiers_can_start_with_digit
-        )
+        dialect_properties = {
+            **{
+                k: v
+                for k, v in vars(klass).items()
+                if not callable(v) and not isinstance(v, classmethod) and not k.startswith("__")
```

Comment:
aren't callable(v) and classmethod the same?

Reply:
That's what I expected too, but for some reason that didn't seem to be the case.

Reply:
Observe:

```python
>>> class A:
...     @classmethod
...     def foo(cls):
...             pass
...     a = 5
...
>>>
>>> {k: v for k, v in vars(A).items() if not callable(v) and not k.startswith("__")}
{'foo': <classmethod(<function A.foo at 0x1011bf130>)>, 'a': 5}
```